src/components/LoginPage/LoginPage.scss

```diff
@@ -0,0 +1,186 @@
+/*
+    24. Login page css
+*/
+.account__login {
```

Comment:
login__account

Reply:
Why? **login__account** is never used in code/ Only **account__login**